### PR TITLE
Mobile payment flow

### LINKED ISSUE #208: Payment flow mobile
## Feature request

Apply mobile mockup from #205 issue

### PR DESCRIPTION
**Pull request by nicola-palermo**
### All Submissions:

*   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
*   [x] Does your code has proper inline documentation? 

### Changes proposed in this Pull Request:

Apply mobile mockup from #205 issue

Closes #208  .

### CODE REVIEW

File: assets/css/stripeStyle.css [hunk 1]
```diff
@@ -266,6 +260,192 @@ table tr {
...
+}
+
+/* Desktop */
+@media screen and (min-width: 1024px) {
+
+	.columns {
+		display: flex;
+		flex-direction: row;
+	}
+
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:312]: Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           

Complexity increasing per file
==============================
- src/stripe/Components/PriceTable.js  1
- src/stripe/Components/InfoPriceColumn.js  1
- src/stripe/Components/Disclaimer.js  1
- src/stripe/Components/SubscribeName.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386&bid=19326916)

File: assets/css/stripeStyle.css [hunk 2]
```diff
@@ -266,6 +260,192 @@ table tr {
...
+	.form-field-container {
+		width: 100%;
+	}
+
+	.summary {
+		border-bottom: 1px rgb(201, 201, 201) solid;
+		overflow: hidden;
+		transition: all .8s cubic-bezier(0.42, 0, 0.15, 1.4);
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:426]: Codacy found an issue: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

File: assets/css/stripeStyle.css [hunk 3]
```diff
@@ -266,6 +260,192 @@ table tr {
...
+	height: 100vh;
+	background: transparent linear-gradient(233deg, #d500f9 0%, #2962ff 99%) 0% 0% no-repeat padding-box;
+	z-index: 100;
+	margin-left: -1rem;
+	padding: 1rem;
+}
+
+.mobile-price-view .nav-back {
+	display: flex;
+    align-items: center;
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:287]: Codacy found an issue: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

File: assets/css/stripeStyle.css [hunk 4]
```diff
@@ -266,6 +260,192 @@ table tr {
...
+
+	.mobile-price-info {
+		display: none;
+	}
+	
+}
+
+/* Tablet */
+@media screen and (max-width: 1023px) {
+	.column {
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:343]: Codacy found an issue: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

File: assets/css/stripeStyle.css [hunk 5]
```diff
@@ -196,41 +185,46 @@ form h2 {
...
+	font: 600 16px/20px "Days One", sans-serif;
+	margin-right: 1rem;
+}
+
 .disclaimer {
 	font: 400 12px/17px "Open Sans", sans-serif;
 }
 
-.disclaimer a {
+.disclaimer a{
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:220]: Codacy found an issue: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 6
           

Complexity increasing per file
==============================
- src/stripe/Components/SubscribeName.js  1
- src/stripe/Components/PriceTable.js  2
- src/stripe/Components/Disclaimer.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386&bid=19326916)

File: assets/css/stripeStyle.css (hunk 3 above)
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:287]: Codacy found an issue: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

File: src/stripe/Components/PriceTable.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+							<span className="cut-line">{/* css only */}</span>
+						</span>
+						0 &euro;
+					</td>
+				)}
+			</tr>
+			<tr className="total">
+				<td>{__('Total', 'wubtitle')}</td>
+				<td className="val">
+					{total} &euro;{' '}
```
Comment (GiuseppeIuculano) [inline src/stripe/Components/PriceTable.js:32]: Codacy found an issue: [Unnecessary block.](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

File: assets/css/stripeStyle.css (hunk 2 above)
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:426]: Codacy found an issue: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

File: assets/css/stripeStyle.css (hunk 5 above)
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:220]: Codacy found an issue: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

File: assets/css/stripeStyle.css (hunk 4 above)
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:343]: Codacy found an issue: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

File: assets/css/stripeStyle.css (hunk 1 above)
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:312]: Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 4
           

Complexity increasing per file
==============================
- src/stripe/Components/PriceTable.js  2
- src/stripe/Components/Disclaimer.js  1
- src/stripe/Components/SubscribeName.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386&bid=19326916)

File: assets/css/stripeStyle.css [hunk 7]
```diff
@@ -266,6 +260,192 @@ table tr {
...
+	}
+
+	.wrapper-form {
+		padding-bottom: 100px;
+	}
+
+	.mobile-price-info {
+		display: block;
+		display: flex;
+    	align-items: center;
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:377]: Codacy found an issue: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

File: assets/css/stripeStyle.css [hunk 8]
```diff
@@ -266,6 +260,192 @@ table tr {
...
+	}
+	
+	.summary .column {
+		width: 100%;
+	}
+
+
+	.summary.closed {
+		height: 80px;
+		transition: all .8s cubic-bezier(0.42, 0, 0.15, 1.4);
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:440]: Codacy found an issue: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

File: assets/css/stripeStyle.css [hunk 9]
```diff
@@ -266,6 +260,192 @@ table tr {
...
+
+	.main.columns {
+		height: 100vh;
+		overflow: hidden;
+	}
+
+	.column {
+		width: 50%;
+	}
+
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:321]: Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

File: assets/css/stripeStyle.css [hunk 10]
```diff
@@ -379,3 +559,7 @@ table tr {
 	position: absolute;
 	transform: rotate(-18deg) translateY(8px);
 }
+
+.as {
+	display: none;
+}
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:565]: Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 3
           

Complexity increasing per file
==============================
- src/stripe/Components/SubscribeName.js  1
- src/stripe/Components/PriceTable.js  2
- src/stripe/Components/Disclaimer.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386&bid=19326916)

File: assets/css/stripeStyle.css (hunk 7 above)
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:377]: Codacy found an issue: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

File: assets/css/stripeStyle.css (hunk 8 above)
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:440]: Codacy found an issue: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

File: assets/css/stripeStyle.css (hunk 9 above)
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:321]: Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- src/stripe/Components/Disclaimer.js  1
- src/stripe/Components/PriceTable.js  2
- src/stripe/Components/SubscribeName.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386&bid=19326916)

File: assets/css/stripeStyle.css [hunk 11]
```diff
@@ -266,6 +260,190 @@ table tr {
...
+	}
+
+	.button-bar {
+		width: 67%;
+	}
+
+	.mobile-price-info {
+		display: none;
+	}
+
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:338]: Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- src/stripe/Components/SubscribeName.js  1
- src/stripe/Components/Disclaimer.js  1
- src/stripe/Components/PriceTable.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386&bid=19326916)

File: assets/css/stripeStyle.css [hunk 12]
```diff
@@ -266,6 +260,189 @@ table tr {
...
+	.column {
+		width: 100%;
+	}
+
+	.summary .columns {
+		display: flex;
+		flex-direction: row;
+	}
+
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:351]: Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- src/stripe/Components/PriceTable.js  2
- src/stripe/Components/SubscribeName.js  1
- src/stripe/Components/Disclaimer.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386&bid=19326916)

File: assets/css/stripeStyle.css [hunk 13]
```diff
@@ -266,6 +260,188 @@ table tr {
...
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:351]: Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- src/stripe/Components/PriceTable.js  2
- src/stripe/Components/Disclaimer.js  1
- src/stripe/Components/SubscribeName.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386&bid=19326916)

File: assets/css/stripeStyle.css [hunk 14]
```diff
@@ -266,6 +260,188 @@ table tr {
...
+	.mobile-price-info {
+		display: block;
+		display: flex;
+		align-items: center;
+	}
+
+	.mobile-price-info .info-icon {
+		margin-left: 10px;
+	}
+
```
Comment (GiuseppeIuculano) [inline assets/css/stripeStyle.css:381]: Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/stripe/Components/PriceTable.js  2
- src/stripe/Components/SubscribeName.js  1
- src/stripe/Components/Disclaimer.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386&bid=19326916)

File: assets/css/stripeStyle.css [hunk 15]
```diff
@@ -252,6 +253,10 @@ table tr {
 	color: #304ffe;
 }
 
+.toggle {
```
Comment (HyprCoder) [inline assets/css/stripeStyle.css:263]: rename this class in is-hidden

File: wubtitle.php [hunk 16]
```diff
@@ -22,7 +22,7 @@
 define( 'WUBTITLE_URL', plugin_dir_url( __FILE__ ) );
 define( 'WUBTITLE_NAME', dirname( plugin_basename( __FILE__ ) ) );
 define( 'WUBTITLE_VER', '0.1.3' );
-$endpoint = 'https://api.wubtitle.com/';
+$endpoint = 'https://jn8vowoh4e.execute-api.eu-west-1.amazonaws.com/dev/';
```
Comment (HyprCoder) [inline wubtitle.php]: This endpoint should be parametized

Comment (HyprCoder) [reply wubtitle.php]: Closing, this is already fixed

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/stripe/Components/Disclaimer.js  1
- src/stripe/Components/SubscribeName.js  1
- src/stripe/Components/PriceTable.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386&bid=19326916)

File: assets/css/stripeStyle.css (hunk 15 above)
Comment (nicola-palermo) [reply assets/css/stripeStyle.css:263]: The toggle should be visible on mobile, the style of that starts at line 406.
I only hide it on desktop breackpoint. Should I change it anyway?

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/stripe/Components/Disclaimer.js  1
- src/stripe/Components/PriceTable.js  2
- src/stripe/Components/SubscribeName.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386&bid=19326916)

**Pull request merged**